aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/AwsXrayRemoteSampler.java

```diff
@@ -148,10 +149,26 @@ private void getAndUpdateSampler() {
   }
 
   private void scheduleSamplerUpdate() {
-    long delay = pollingIntervalNanos + RANDOM.nextInt(jitterNanos);
+    @SuppressWarnings(
+        "OptionalGetWithoutIsPresent") // getAsLong: RANDOM.longs should always provide a value, so
+    // we can do an unchecked unwrap here
+    long delay = pollingIntervalNanos + RANDOM.longs(0, jitterNanos).findFirst().getAsLong();
```

Comment:
Let's replace the `jitterNanos` field with `RANDOM.longs(0, jitterNanos)` to not construct the stream every time.

Reply:
Sorry, I am not very used to working with streams. How would I get the next value from a stream? From reading examples and the documentation a little streams aren't supposed to use that way?